frontend/mr-admin-flate/src/components/tabell/TiltaksgjennomforingsTabell.tsx

```diff
@@ -122,147 +118,163 @@ export const TiltaksgjennomforingsTabell = ({ skjulKolonner }: Props) => {
 
   return (
     <div className={styles.tabell_wrapper}>
-      <PagineringsOversikt
-        page={page}
-        antall={tiltaksgjennomforinger.length}
-        maksAntall={pagination?.totalCount}
-        type="tiltaksgjennomføringer"
-        antallVises={filter.antallGjennomforingerVises}
-        setAntallVises={(size) => {
-          resetPaginering(setPage);
-          setFilter({ ...filter, antallGjennomforingerVises: size });
-        }}
-      />
-
-      <Table
-        sort={sort!}
-        onSortChange={(sortKey) => handleSort(sortKey!)}
-        className={styles.tabell}
-      >
-        <Table.Header>
-          <Table.Row className={styles.tiltaksgjennomforing_tabellrad}>
-            {headers
-              .filter((header) => {
-                return skjulKolonner ? !skjulKolonner[header.sortKey] : true;
-              })
-              .map((header) => (
-                <Table.ColumnHeader
-                  key={header.sortKey}
-                  sortKey={header.sortKey}
-                  sortable={header.sortable}
-                  style={{ width: header.width }}
-                >
-                  {header.tittel}
-                </Table.ColumnHeader>
-              ))}
-          </Table.Row>
-        </Table.Header>
-        {tiltaksgjennomforinger.length > 0 ? (
-          <Table.Body>
-            {tiltaksgjennomforinger.map((tiltaksgjennomforing, index) => {
-              return (
-                <Table.Row
-                  key={index}
-                  className={styles.tiltaksgjennomforing_tabellrad}
-                >
-                  <SkjulKolonne skjul={!!skjulKolonner?.navn}>
-                    <Table.DataCell
-                      aria-label={`Navn på tiltaksgjennomforing: ${tiltaksgjennomforing.navn}`}
-                      className={styles.title}
-                    >
-                      <Lenke
-                        to={`/tiltaksgjennomforinger/${tiltaksgjennomforing.id}`}
-                        data-testid="tiltaksgjennomforingrad"
+      <div>
```

Comment:
Trengs denne diven?

Reply:
Ja, for å legge checkbox for mine gjennomføringer ved siden av dropdown for antall